<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivitiesForm_template</class>
 <widget class="QDialog" name="ActivitiesForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="This is the title of the activities dialog">Activities dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="activeTextLabel">
         <property name="text">
          <string notr="true">No: 1000 / 1000</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="totalTextLabel">
         <property name="text">
          <string notr="true">Dur: 1000 / 1000</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QListWidget" name="activitiesListWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="activityDescriptionTextLabel">
         <property name="text">
          <string>Current activity</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="activityTextEdit"/>
       </item>
       <item>
        <widget class="QGroupBox" name="filterGroupBox">
         <property name="title">
          <string>Filter</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="teachersTextLabel">
            <property name="text">
             <string>Teacher</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="teachersComboBox"/>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="studentsTextLabel">
              <property name="text">
               <string comment="It refers to students sets, but I want to keep the string short">Students</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="recursiveCheckBox">
              <property name="text">
               <string>Show related</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QComboBox" name="studentsComboBox"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="subjectsTextLabel">
            <property name="text">
             <string>Subject</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QComboBox" name="subjectsComboBox"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="activityTagsTextLabel">
            <property name="text">
             <string>Activity tag</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QComboBox" name="activityTagsComboBox"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addActivityPushButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modifyActivityPushButton">
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeActivityPushButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="commentsPushButton">
       <property name="text">
        <string>Comments</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpPushButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
